slides/snippets/src/main/java/CreateSlide.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START slides_create_slide]
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.slides.v1.Slides;
+import com.google.api.services.slides.v1.SlidesScopes;
+import com.google.api.services.slides.v1.model.BatchUpdatePresentationRequest;
```

Comment:
Looks like you are importing `model.*` in some cases, and them classes individually in others. Is that due to the IDE preferring explicit imports? Could we not just import slides.v1.* for all of these? Can't imagine that's a terribly large amount of extra overhead.

Reply:
Yep, just an oversight. Preference is towards explicit imports